sspl_test/run_test.py

```diff
@@ -28,13 +28,11 @@
 import errno
 import re
 import argparse
-from generate_test_report import generate_html_report
-from framework.utils.config_reader import ConfigReader
-
 # Adding sspl and sspl_test path
 test_path = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 os.sys.path.append(os.path.join(test_path))
-
+from sspl_test.generate_test_report import generate_html_report
```

Comment:
all imports should be before any code

Reply:
Whenever I use sys.path.append, the new directory will be added. However, once I close python, the list will revert to the previous (default?) values. 
I am working on how we can permanently add a '/opt/seagate/cortx/sspl/low-level' to PYTHONPATH.

Reply:
check similar previous comment

Reply:
I Will address this comment separately with https://jts.seagate.com/browse/EOS-16240 ticket.